flink-table/flink-sql-client/src/test/resources/sql-client-help-command.out

```diff
@@ -1,23 +1,19 @@
-CLEAR		Clears the current terminal.
-CREATE TABLE		Create table under current catalog and database.
-DROP TABLE		Drop table with optional catalog and database. Syntax: "DROP TABLE [IF EXISTS] <name>;"
-CREATE VIEW		Creates a virtual table from a SQL query. Syntax: "CREATE VIEW <name> AS <query>;"
-DESCRIBE		Describes the schema of a table with the given name.
-DROP VIEW		Deletes a previously created virtual table. Syntax: "DROP VIEW <name>;"
-EXPLAIN		Describes the execution plan of a query or table with the given name.
+The following commands are available:
+
 HELP		Prints the available commands.
+QUIT/EXIT		Quits the SQL CLI client.
+CLEAR		Clears the current terminal.
+SET		Sets a session configuration property. Syntax: "SET '<key>'='<value>';". Use "SET;" for listing all properties.
```

Comment:
One more nit-picking :) 
How about we left-justify all the the explanation sentences, such that the whole page appears as 2 columns? I think if they are all aligned they'll be easier to read